benchmarks/src/main/java/io/crate/execution/engine/aggregation/GroupingLongCollectorBenchmark.java

```diff
@@ -108,7 +143,17 @@ public void createGroupingCollector() throws Exception {
             DataTypes.INTEGER
         );
         var memoryManager = new OnHeapMemoryManager(bytes -> {});
-        groupBySumCollector = createGroupBySumCollector(sumAgg, memoryManager);
+
+        currentNettyMapCollector = createGroupBySumCollector(sumAgg, memoryManager, () -> new PrimitiveMapWithNulls(new LongObjectHashMap<>()));
+        jdkCollector = createGroupBySumCollector(sumAgg, memoryManager, () -> new HashMap());
+        kolobokeCollector = createGroupBySumCollector(sumAgg, memoryManager, () -> new PrimitiveMapWithNulls(HashLongObjMaps.newMutableMap()));
+        fastutilCollector = createGroupBySumCollector(sumAgg, memoryManager, () -> new PrimitiveMapWithNulls(new Long2ObjectOpenHashMap<>()));
```

Comment:
Just so be sure - these maps don't support null values natively?

Reply:
Koloboke can support null keys but for that I would have to generate it - https://github.com/leventov/Koloboke/blob/master/compile/tutorial.md#allow-the-null-key - not sure it worth it as current Netty map even with wrapper `PrimitiveMapWithNulls` overhead beats them all

FastUtil - they have smth like `defaultReturnValue` which I guess bears same logic - I wrapped with our class for simplicity but will give native null a try